net-vpn/mullvad-bin/mullvad-bin-2022.5.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit rpm xdg-utils
+MY_PN="mullvad-bin"
+DESCRIPTION="Official Mullvad VPN CLI for Linux"
+HOMEPAGE="https://mullvad.net"
+SRC_URI="https://github.com/mullvad/mullvadvpn-app/releases/download/${PV}/MullvadVPN-${PV}_x86_64.rpm"
+LICENSE="GPL-3"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+DEPEND=""
+RDEPEND="${DEPEND}"
+S="${WORKDIR}"
+RESTRICT="strip"
+
+src_unpack(){
+rpm_src_unpack "${A}"
```

Comment:
This phase shouldn't be necessary at all? `rpm_src_unpack` should be called automatically.